<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsItem</class>
 <widget class="QWidget" name="SettingsItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelDirectories">
     <property name="text">
      <string>Directories:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editDirectories"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonDirectory">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelExtensions">
     <property name="text">
      <string>Extensions:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="editExtensions"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxGenerateIndex">
     <property name="text">
      <string>Generate index</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelIndexLocation">
     <property name="text">
      <string>Index location:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="editIndexLocation"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="buttonBrowseIndexLocation">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="buttonUpdateNow">
     <property name="text">
      <string>Update now</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxGenerateIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>editIndexLocation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxGenerateIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonUpdateNow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxGenerateIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonBrowseIndexLocation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
